src/chewingio.c

```diff
@@ -2352,3 +2352,14 @@ CHEWING_API int chewing_clean_bopomofo_buf(ChewingContext *ctx)
     MakeOutput(pgo, pgdata);
     return 0;
 }
+
+CHEWING_API int chewing_phone_to_bopomofo( unsigned short phone, char *buf, unsigned short len )
+{
```

Comment:
Please follow the consistent coding style as well. You can eliminate one "else" by adjusting the condition logic.